test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -593,6 +593,26 @@ def func(tensor):
         tensor = common_utils.get_whitenoise(sample_rate=44100)
         self._assert_consistency(func, tensor)
 
+    def test_character_edit_distance(self):
+
+        def func(tensor):
+            ref = "abc"
+            hyp = "bcd"
+            return F.character_edit_distance(ref, hyp)
+
+        dummy = torch.zeros(1, 1)
+        self._assert_consistency(func, dummy)
```

Comment:
Please do not use `_assert_consistency` here.
It is not designed for functions that do not accept tensor.
Please implement the logic directly in the test method.
Also instead of hard coding `ref` and `hyp`, make them function argument in `func`.

Reply:
Thanks for the suggestion, I've fixed them in the new commit.